src/js/angular/explore/fit-text.directive.js

```diff
@@ -0,0 +1,98 @@
+angular
+    .module('graphdb.framework.explore.directives.fittext', [])
```

Comment:
Is this directive related with the `explore` or it's a generic one? If it's generic isn't it better to move it core/directives?

Reply:
I agree - will be better to be generic one.